### PR TITLE
Polygon amoy support

### PR DESCRIPTION
**Pull request by Parth2412**
- Added support for polygon amoy testnet.

### CODE REVIEW

**Comment by Parth2412** [comment]
Covering Issue #3379

**Comment by Parth2412** [comment]
Please review: @acolytec3 @steveluscher @dbrock @Arachnid @axic

**Comment by Parth2412** [comment]
@holgerd77 Please review

File: packages/common/test/customChains.spec.ts [hunk 1]
```diff
@@ -98,6 +98,52 @@ describe('[Common]: Custom chains', () => {
     }
   })
```
Comment (acolytec3) [inline packages/common/test/customChains.spec.ts:100]: Did you make any substantial changes in this new test other than change the custom chain reference on line 106 to `PolygonAmoy`?  I don't see anything obvious so not sure we need a duplicate of the test above with a different chain name.

File: packages/tx/README.md [hunk 2]
```diff
@@ -364,6 +364,7 @@ The following L2 networks have been tested to work with `@ethereumjs/tx`, see us
 | xDai Chain               |  `Common.xDaiChain`                   |  [#1323](https://github.com/ethereumjs/ethereumjs-monorepo/issues/1323) |
 | Optimistic Kovan         | `Common.OptimisticKovan`              | [#1554](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1554)    |
 | Optimistic Ethereum      | `Common.OptimisticEthereum`           | [#1554](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1554)    |
+| Polygon Amoy Testnet     | `CustomChain.PolygonAmoy`             | [#3379](https://github.com/ethereumjs/ethereumjs-monorepo/issues/3379)  |
```
Comment (acolytec3) [inline packages/tx/README.md:367]: Did you test and confirm that transactions created with our libraries are accepted and included in Amoy blocks?

**Review by acolytec3 — COMMENTED** [review]
Thanks for the PR.  Left a couple of questions to address before we can merge.

**Comment by acolytec3** [comment]
@Parth2412 just checking in, are you able to address the review questions?

**Comment by acolytec3** [comment]
Going to close for now as no follow-up.

**Pull request closed without merging**